src/components/errors/unauthorized-error.tsx

```
@@ -0,0 +1,44 @@
+  Card,
+  CardHeader,
+  CardTitle,
+  CardDescription,
+  CardContent,
+} from "@/components/ui/card";
+
+interface UnauthorizedErrorProps {
+  onRetry?: () => void;
+}
+
+export function UnauthorizedError({}: // onRetry
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)